src/docs/development/add-to-app/android/project-setup.md

```diff
@@ -202,7 +202,7 @@ repositories {
     // if using a relative path.
   }
   maven {
-    url 'http://download.flutter.io'
+    url 'https://storage.googleapis.com/download.flutter.io'
```

Comment:
sad loss of vanity url, but seems sensible. I have a feeling this string exists in more places such as flutter/samples etc.

Reply:
yup https://github.com/flutter/samples/blob/master/add_to_app/android_using_prebuilt_module/app/build.gradle#L36